src/test/scala/com/databricks/spark/avro/AvroSuite.scala

```diff
@@ -394,4 +394,93 @@ class AvroSuite extends FunSuite {
     assert(newDf.count == 8)
   }
 
+  test("test doc in meta") {
+    val df = TestSQLContext.load(episodesFile, "com.databricks.spark.avro")
+    df.schema.fields(0).metadata.getString(SchemaConverters.METADATA_KEY_DOC)
+
+    for (x <- df.schema.fields) {
+      if (x.name == "title") {
+        assert("episode title" == x.metadata.getString(SchemaConverters.METADATA_KEY_DOC))
+      } else if (x.name == "doctor") {
+        assert("main actor playing the Doctor in episode" ==
+          x.metadata.getString(SchemaConverters.METADATA_KEY_DOC))
+      } else if (x.name == "air_date") {
+        assert("initial date" == x.metadata.getString(SchemaConverters.METADATA_KEY_DOC))
+      }
+    }
+  }
+
+  test("test aliases in meta") {
+    val df = TestSQLContext.load(testFile, "com.databricks.spark.avro")
+
+    for (x <- df.schema.fields) {
```

Comment:
I guess people will believe the metadata are not type dependent. Hence you can put them only on one field in test file - let's say "string" and then this whole test can become
```scala
val df = TestSQLContext.load(testFile, "com.databricks.spark.avro")
assert(df.schema("string").getStringArray(SchemaConverters.METADATA_KEY_ALIASES) === Array("string_alias1", "string_alias1")
```
Would that make it easier?